### ISSUE TITLE
It seems that all numbers the model can't recognise is output 4

### ISSUE THREAD

**Issue by mobyIsMe** [hwchong/MNIST_DRAW#2]
such as input 10 or other numbers which are not regular written， it outputs 4，input 11 ， it output 4 sometimes and 1 sometimes. The model doesn't support numbers recognition above 10?

**Comment by hwchong** [comment]
No, MNIST (the dataset) doesn't have double digits. The model is flexible and you can always retrain it on a dataset with double digits, just make sure you have enough logits in your softmax function output.
